src/css/styles.scss

```diff
@@ -94,12 +94,12 @@ main {
   outline: 0;
   width: 30%;
     &:hover {
-    cursor: pointer;
+    // cursor: pointer;
     box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
   }
-    &:active {
-      transform: scale(1.03);
-    }
+    // &:active {
```

Comment:
Good call on commenting this stuff out - I messed around with it and it definitely does not work to have the scale on click since it scales everytime you try and fill out an input. Something that might work is if we put the scale on hover and then it will stay scaled as long as they are filling out the form - that might make that UI cleaner.